src/dex-ui/services/HederaService/HederaService.ts

```diff
@@ -99,11 +105,11 @@ function createHederaService() {
     } = addLiquidityDetails;
 
     // TODO: remove fallbacks if no signer and not all details are provided
-    const firstTokenAddr = firstTokenAddress ? firstTokenAddress : tokenA;
-    const secondTokenAddr = secondTokenAddress ? secondTokenAddress : tokenB;
+    const firstTokenAddr = firstTokenAddress;
```

Comment:
We can use `firstTokenAddress` and `secondTokenAddress` directly.

Reply:
Done